apps/src/code-studio/header.js

```diff
@@ -168,10 +171,42 @@ function setupReduxSubscribers(store) {
 setupReduxSubscribers(getStore());
 
 function setUpGlobalData(store) {
-  store.dispatch(asyncLoadUserData());
+  fetch('/api/v1/users/current')
```

Comment:
Make sure the Infra team is okay with the increased traffic. This is going to add an HTTP with every page load from now on, right?